runs/updateConfig.sh

```diff
@@ -1115,6 +1115,7 @@ updateConfig(){
 		echo "pv1_idc=1" >> $ConfigFile
 		echo "pv1_idd=1" >> $ConfigFile
 		echo "speicher1_ip=192.168.0.17" >> $ConfigFile
+		echo "speicher1_ip2=192.168.0.17" >> $ConfigFile
```

Comment:
Dieser Eintrag wird nie ergänzt werden, da in einer vorherigen Version bereits `bezug1_ip` vorhanden war. Bitte in einem eigenen IF ergänzen, falls das wirklich notwendig ist (siehe anderer Kommentar).